### PR TITLE
Addressing VCF 4.4 feedback

### PR DESCRIPTION
**Pull request by d-cameron**
Incorporating feedback from public comment period

### CODE REVIEW

File: VCFv4.4.draft.tex [hunk 1]
```diff
@@ -188,21 +188,26 @@ \subsubsection{Alternative allele field format} \label{altfield}
...
   \item CNV Copy number variable region (may be both deletion and duplication)
-  \item BND Breakend
 \end{itemize}
 
-The CNV category should not be used when a more specific category can be applied. Reserved subtypes include:
+The CNV category should not be used when a more specific category can be applied.
+
+Implementations are free to define their own subtypes.
+The presence of a subtype does not change either the copy number or breakpoint interpretation of a symbolic structural variant allele.
+The following subtypes are recommended:
```
[the remark this answers is not in the document]
Comment (d-cameron) [reply VCFv4.4.draft.tex:202]: It's a clarification that subtype can't change. E.g. a DUP breakpoint is a tandem-duplication-like breakpoint. if a caller makes a custom subtype such as `DUP:DISPERSED`, that would change the breakpoints associated with the rearrangement. This text clarifies that it is valid for tools to infer breakpoints and CN from just the top-level type.

Implementations are free to make their own subtypes, they just can't change the meaning.

File: examples/vcf/sv44.vcf [hunk 2]
```diff
@@ -0,0 +1,26 @@
+##fileformat=VCFv4.4
```
[the remark this answers is not in the document]
Comment (d-cameron) [reply examples/vcf/sv44.vcf:1]: As a new copy/symlink or moved there? Like `examples/vcf/simple.vcf`, the VCF content is included in the specifications document itself, so I put it in the same location as simple.vcf.

Should we move simple.vcf as well?

[the remark this answers is not in the document]
Comment (tcezard) [reply examples/vcf/sv44.vcf:1]: Sorry I thought this was in `test`. It is fine where it is.

[the remark this answers is not in the document]
Comment (tcezard) [reply examples/vcf/sv44.vcf:1]: Although we should also build new test files that are passing/failing the new specs changes.

**Pull request merged**